scripts/compute_speaker_stats.py

```diff
@@ -0,0 +1,110 @@
+import argparse
```

Comment:
Are the files under scripts needed for this PR? It seems like a lot of dev code

Reply:
If they are important, maybe we can move them to "scripts/dataset_processing/tts/ssl"?

Reply:
removed scripts for evaluation. Now there are two left - one for performing voice conversion from a trained model (ssl_tts_vc.py) and the other for preparing data for fastpitch training (make_sup_data.py). Let me know where these should be placed.